tfx/examples/cifar10/distributed/README.md

```diff
@@ -0,0 +1,86 @@
+# CIFAR-10 Transfer Learning and MLKit integration Example with Distributed Training
+
+This example illustrates how to modify the base example for distributed training with a distributed node pool on Google Kubernetes Engine(GKE).
```

Comment:
Add space between Engine and (GKE)

Reply:
Please address @chuanyu's comments throughout.